src/test/components/TransformShortcuts.test.js

```diff
@@ -122,21 +125,21 @@ function setupStore(childrenToRender) {
     profile,
     funcNamesDictPerThread: [funcNames],
   } = getProfileFromTextSamples(`
-    A               A               A
+    A           A           A
     B[lib:XUL]  B[lib:XUL]  B[lib:XUL]
     B[lib:XUL]  B[lib:XUL]  B[lib:XUL]
     B[lib:XUL]  B[lib:XUL]  B[lib:XUL]
-    C               C               H
-    D               F               I
-    E               E
+    C           C           H
+    D           F           I
+    E           E
```

Comment:
Oops, this was producing a completely bad profile ! 😧

Indeed, the first line is where we take the columns from... The other lines aren't so important, but the first line is primordial.

Reply:
I thought we were only splitting them by their 2+ spaces and then getting the columns there. TIL!